doc/CONTRIBUTING.md

```diff
@@ -129,14 +128,7 @@ modified code when you run these commands.
 <a id="Testing_with_coverage_analysis"></a>
 #### Testing with coverage analysis
 
-Coverage analysis is only enabled for `src/core/*` for now. To generate coverage
-reports, first pin Bisect_ppx to its development version:
-
-```
-opam pin add --dev-repo bisect_ppx
-```
-
-Then run
+To generate coverage reports, run
```

Comment:
minor nit, missing `:` at end of sentence

Reply:
Ultra grammar mode! :p

Modulo mistakes made here and there, I put the colon (sometimes preceded by "with" or some other preposition) when the sentence introducing the quote/list/other block can stand on its own, and the quote, etc., can be viewed as a separate illustration. It should be possible to replace the colon with a period, maybe deleting a preposition or other filler word.

I tend to omit the colon when the introduction sentence doesn't make sense on its own. In this case, it has a *different* sense:

> "To generate coverage reports, run."
>
> ![whenever-i-go-running-running-meme 1](https://user-images.githubusercontent.com/12073668/37879797-e2a22a20-3043-11e8-84bc-da6584483aa6.png)

The command, in this case, cannot be taken out of that sentence (or phrase), as it's the direct object. It should actually be set inline, but is a block for easier copy/pasting when someone just wants to quickly find all the commands in sequence, and avoid reading text.

I believe this is the recommended usage, at least in the U.S.